src/bilder/images/dagster/deploy.py

```diff
@@ -63,9 +62,11 @@
         "CONSUL_TEMPLATE_VERSION", CONSUL_TEMPLATE_VERSION
     ),
     "vault": os.environ.get("VAULT_VERSION", VAULT_VERSION),
-    "dagster": os.environ.get("DAGSTER_VERSION", DAGSTER_VERSION),
 }
 
+DOCKER_REPO_NAME = os.environ.get("DOCKER_REPO_NAME", "mitodl/mono-dagster")
+DOCKER_IMAGE_DIGEST = os.environ.get("DOCKER_IMAGE_DIGEST")
```

Comment:
Maybe add a fallback to fetch the image hash from the repo if the env var isn't set?

Reply:
So that is more than I can figure out in a moment due to `@` vs `:` when using digest vs tags but definitely obtainable.